### PR TITLE
dev-libs/hyperscan: Version bump 5.1.0

### PR DESCRIPTION
**Pull request by okias**
Closes: https://github.com/gentoo/gentoo/pull/9863
Closes: https://bugs.gentoo.org/674220

Original author: Christian Rößner
Signed-off-by: David Heidelberg <david@ixit.cz>

### CODE REVIEW

File: dev-libs/hyperscan/hyperscan-5.0.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="cpu_flags_x86_ssse3 static-libs"
+
+RDEPEND="
+	${PYTHON_DEPS}
+	dev-util/ragel
+	dev-libs/boost
```
Comment (hydrapolic) [inline dev-libs/hyperscan/hyperscan-5.0.0.ebuild]: Maybe adding a slot operator to rebuild this when boost is upgraded?

**Comment by okias** [comment]
Month passed away, any progress on this?

**Comment by a17r** [comment]
@croessner are you still there?

File: dev-libs/hyperscan/hyperscan-5.0.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 )
+
+inherit cmake-utils python-r1
+
+DESCRIPTION="High-performance regular expression matching library"
+SRC_URI="https://github.com/01org/hyperscan/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+HOMEPAGE="https://01.org/hyperscan"
```
Comment (a17r) [inline dev-libs/hyperscan/hyperscan-5.0.0.ebuild]: `HOMEPAGE` is supposed to be next after `DESCRIPTION`, `SRC_URI` below.

File: dev-libs/hyperscan/hyperscan-5.0.0.ebuild [hunk 3]
```diff
@@ -0,0 +1,35 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```
Comment (a17r) [inline dev-libs/hyperscan/hyperscan-5.0.0.ebuild]: nothing's stopping you from bumping this to `EAPI-7` right away.

**Comment by okias** [comment]
done :)

**Comment by croessner** [comment]
@a17r I am alive 😊

**Comment by a17r** [comment]
@croessner and do you approve of this PR?

**Comment by croessner** [comment]
Yes perfect 👍

**Comment by okias** [comment]
@a17r I cannot get rid of "Changes requested" can you merge it anyway?

**Pull request closed without merging**